spec/02-integration/09-hybrid_mode/11-status_spec.lua

```diff
@@ -107,63 +104,114 @@ for _, strategy in helpers.each_strategy() do
-
-          local res = http_client:send({
-            method = "GET",
-            path = "/status/ready",
-          })
-
-          local status = res and res.status
-          http_client:close()
-          if status == 200 then
-            return true
-          end
-        end, 10)
-
-        -- recovery state between tests
-        assert(start_kong_cp())
-
-        helpers.wait_until(function()
-          local http_client = helpers.http_client('127.0.0.1', dp_status_port)
-
-          local res = http_client:send({
-            method = "GET",
-            path = "/status/ready",
-          })
-
-          local status = res and res.status
-          http_client:close()
-          if status == 200 then
-            return true
-          end
-        end, 10)
-
-      end)
+      if rpc_sync == "off" then
+        it("should return 200 on data plane after configuring", function()
+          helpers.wait_until(function()
+            local http_client = helpers.http_client('127.0.0.1', dp_status_port)
+
+            local res = http_client:send({
+              method = "GET",
+              path = "/status/ready",
+            })
+
+            local status = res and res.status
+            http_client:close()
+            if status == 200 then
+              return true
+            end
+          end, 10)
+
+          assert(helpers.stop_kong("serve_cp", nil, nil, "QUIT", false))
+
+          -- DP should keep return 200 after CP is shut down
+          helpers.wait_until(function()
+
+            local http_client = helpers.http_client('127.0.0.1', dp_status_port)
+
+            local res = http_client:send({
+              method = "GET",
+              path = "/status/ready",
+            })
+
+            local status = res and res.status
+            http_client:close()
+            if status == 200 then
+              return true
+            end
+          end, 10)
+
+          -- recovery state between tests
+          assert(start_kong_cp())
+
+          helpers.wait_until(function()
+            local http_client = helpers.http_client('127.0.0.1', dp_status_port)
+
+            local res = http_client:send({
+              method = "GET",
+              path = "/status/ready",
+            })
+
+            local status = res and res.status
+            http_client:close()
+            if status == 200 then
+              return true
+            end
+          end, 10)
+        end)
+      end
     end)
 
+    if rpc == "on" and rpc_sync == "on" then
+      describe("dp status ready when rpc_sync == on", function()
+        lazy_setup(function()
+          assert(start_kong_cp())
+          assert(start_kong_dp())
+        end)
+
+        lazy_teardown(function()
+            assert(helpers.stop_kong("serve_cp"))
+            assert(helpers.stop_kong("serve_dp"))
+        end)
+
+        it("should return 200 on data plane after configuring when rpc_sync == on ", function()
```

Comment:
```suggestion
        it("should return 200 on data plane after configuring when rpc_sync == on", function()
```

Please note this detail, thank you! In Kong, a strict coding style is considered friendly. Otherwise, other developers will decide whether they need to notify you about it.